src/app/components/tag/TagInput.js

```diff
@@ -8,6 +8,17 @@ import CheckContext from '../../CheckContext';
 import { units, Row } from '../../styles/js/shared';
 import { createTag } from '../../relay/mutations/CreateTagMutation';
 import { stringHelper } from '../../customHelpers';
+import { makeStyles } from '@material-ui/core/styles';
+
+const Box = styled.div`
+p={2}
+`;
+
+
+const Button = styled.button`
+ marginLeft: 'auto';
+  `;
```

Comment:
This shouldn't be a `styled.button`. We need to still use the `material-ui` `Button`.